src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1151,4 +1151,7 @@ This version used in: {2}</value>
   <data name="Add_document_0" xml:space="preserve">
     <value>Add document '{0}'</value>
   </data>
+  <data name="Flow_analysis_failed" xml:space="preserve">
+    <value>Flow analysis failed for the selection.</value>
```

Comment:
Super dn't like this error message.  Can we not get enough information to say "selection uses a local function and cannot be extracted."?